contrib/test/ci/vars.yml

```diff
@@ -169,6 +164,11 @@ kata_skip_pod_tests:
   - 'test "systemd cgroup_parent correctly set"'
   - 'test "kubernetes pod terminationGracePeriod passthru"'
   - 'test "disable crypto.fips_enabled when FIPS_DISABLE is set"'
+  - 'test "run container with memory_limit_in_bytes -1"'
+  - 'test "run container with memory_limit_in_bytes 12.5MiB"'
+  - 'test "run container with container_min_memory 17.5MiB"'
+  - 'test "run container with container_min_memory 5.5MiB"'
+  - 'test "run container with empty container_min_memory"'
```

Comment:
An idea (perhaps to follow up later or not):

Do we want to include a test that uses the minimal viable memory size for crun? Just to ensure that it can actually start with that low memory limit? Now, and going forward, that is...

Thoughts?